pipeline/merger/README.md

```diff
@@ -0,0 +1,25 @@
+# Merger
+
+Merging rules are applied per field, with a matching rule being applied to a single `target` (a Sierra work) and one or more`sources`.
```

Comment:
> to a single target (a Sierra work) and one or moresources.

Are we still of the mind that the target is always Sierra?

(space missing too)